src/components/MembershipTypeButton.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+
+import React from "react";
+
+interface MembershipOptionProps {
+  selectedMembership: boolean | undefined;
+  onClick: (isMember: boolean) => void;
+  typeHeading: string;
+  description1: string;
+  description2: string;
+}
+
+export const MembershipTypeButton: React.FC<MembershipOptionProps> = ({
+  selectedMembership,
+  onClick,
+  typeHeading,
+  description1,
+  description2,
+}) => (
+  <label
+    className={`flex flex-col py-[21px] px-8 mb-[24px] h-32 shadow rounded hover:pointer cursor-pointer ${
+      selectedMembership
```

Comment:
Also, maybe instead of having a fixed height, you could remove that and set padding to p-6? So that the text is centered and doesn't overflow in smaller screens